presto-main/src/main/java/io/prestosql/sql/planner/plan/ApplyNode.java

```diff
@@ -144,10 +153,7 @@ public Node getOriginSubquery()
     @JsonProperty("outputSymbols")
     public List<Symbol> getOutputSymbols()
     {
-        return ImmutableList.<Symbol>builder()
-                .addAll(input.getOutputSymbols())
-                .addAll(subqueryAssignments.getOutputs())
```

Comment:
@raunaqmorarka after second thought i think ApplyNode case is different from LateralJoinNode case.
In ApplyNode case, I see no possibility that `input.getOutputSymbols()` have non empty intersection with `subqueryAssignments.getOutputs()`.
Since they cannot intersect, there should be no need to remove duplicates (`input.getOutputSymbols()` and `subqueryAssignments.getOutputs()` shall not have duplicates internally).

I would suggest to revert changes in ApplyNode.
Before you do that, I'd like to hear @kokosing's and @martint's thoughts.

Reply:
There could be a case where output of subquery would be a correlation. In such case `input.getOutputSymbols()` and `subqueryAssignments.getOutputs()` might have common symbols.

Reply:
eg.
```
SELECT col, (SELECT col) FROM (VALUES 1, 2) t(col)
```

Reply:
@kokosing will this create ApplyNode or a  LateralJoinNode?

Reply:
@findepi pointed out that the query above produces `LateralJoinNode` (see io.prestosql.sql.planner.plan.ApplyNode#isSupportedSubqueryExpression)

Lets change only LateralNode

Reply:
Ok, I've updated the PR to have only the LateralJoinNode change